lib/iris/tests/unit/fileformats/pp_rules/test__convert_scalar_vertical_coords.py

```diff
@@ -161,6 +161,34 @@ def test_cross_section(self):
         self._check_soil_level(_lbcode(ix=1, iy=2), expect_match=False)
 
 
+class TestLBVC006_SoilDepth(TestField):
+    def _check_soil_level(self, lbcode, expect_match=True):
```

Comment:
Dah! I only changed the equivalent in `test__convert_vertical_coords.py` :scream: